adsws/modules/oauth2server/views/settings.py

```diff
@@ -59,7 +59,7 @@ def wrapper(f):
         @wraps(f)
         def decorated(*args, **kwargs):
             if 'client_id' not in kwargs:
-                abort(500)
+                abort(500, message='client_id not found in args: {}'.format(kwargs))
```

Comment:
This will return the message to the client but the message is not understandable by external users. But still, the intention is good since we want to distinguish between error. I suggest we use the same message but with something like:

```
current_app.logger.error("Aborting with status code 500 due to....")
```

Right before the abort. Also, you'll need to import `from flask import current_app`. Then we will be able to see this in graylog when searching the `trace_id` of the request.